### PR TITLE
Nbo/add tooltips to enum prod 11209

### PR DESCRIPTION
**Pull request by nborde-CSM**

### CODE REVIEW

File: src/inputs/BasicInputs/BasicEnumInput/BasicEnumInput.js [hunk 1]
```diff
@@ -42,13 +42,29 @@ export const BasicEnumInput = (props) => {
...
           {enumValues.map((option) => (
-            <MenuItem key={option.key} value={option.key} data-cy={option.key}>
-              {option.value}
+            <MenuItem
+              key={option.key}
+              value={option.key}
+              data-cy={option.key}
+              sx={{ p: '0px' }} // Remove default MenuItem padding to increase the hover area for the items tooltips
+            >
+              <Tooltip
```
Comment (esasova) [inline src/inputs/BasicInputs/BasicEnumInput/BasicEnumInput.js:55]: can we use our `FadingTooltip` component here or it is not suitable for Selects ?

Comment (csm-thu) [reply src/inputs/BasicInputs/BasicEnumInput/BasicEnumInput.js:55]: I tried using it, but our `FadingTooltip` component adds an intermediary `<div>` that breaks the layout and the tooltip behavior somehow. I didn't take the time to look further into this issue, but we may have to add an optional prop to the `FadingTooltip` to declare whether or not the `<div>` should be added automatically.

Comment (nborde-CSM) [reply src/inputs/BasicInputs/BasicEnumInput/BasicEnumInput.js:59]: I confirm that I tried to use FadingTooltip and it doesn' t work correctly

**Review by esasova — APPROVED** [review]
LGTM :+1:

**Pull request merged**